### PR TITLE
Fix(Dependencies): Add testcafe

### PR DESCRIPTION
**Pull request by samuellye98**
Add open-attestation and testcafe so users can run their own integration tests.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,13 +10,16 @@
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
     "react-pdf": "^4.1.0",
-    "react-scripts": "3.0.1"
+    "react-scripts": "3.0.1",
+    "@govtechsg/open-attestation": "^1.1.30"
```
Comment (yehjxraymond) [inline package.json]: Why is this added?

Comment (yehjxraymond) [reply package.json]: The document renderer does not need to know of the OA format

Comment (samuellye98) [reply package.json]: For the integration tests:
import { getData } from "@govtechsg/open-attestation";
...
const Certificate = "./Ropsten-Demo.json";
test("Govtech Demo certificate is rendered correctly", async t => {
  const certificateContent = getData(
    JSON.parse(readFileSync(join(__dirname, Certificate)).toString())
  );
  await t.eval(
    () => window.openAttestation.renderDocument(certificateContent),
    {
      dependencies: { certificateContent }
    }
  );
...

Comment (yehjxraymond) [reply package.json]: You can instead of using the opencert file use the content of the opencert file.

Comment (samuellye98) [reply package.json]: yeap i thought of that but wouldnt institutions normally put their unwrapped opencerts file in the same folder and they may find it difficult to unwrap their opencerts file themselves?

Comment (yehjxraymond) [reply package.json]: They will have that that already to get the batched ones. It's easier for them to read and write their templates as well.
